mars/dataframe/execution/tests/test_arithmetic_execution.py

```diff
@@ -226,6 +227,35 @@ def testAddWithAdded(self):
 
         pd.testing.assert_frame_equal(expected, result)
 
+    def testRadd(self):
+        data1 = pd.DataFrame(np.random.rand(10, 10))
+        df1 = from_pandas(data1, chunk_size=5)
+        data2 = pd.DataFrame(np.random.rand(10, 10))
+        df2 = from_pandas(data2, chunk_size=6)
+        radd = getattr(df2, '__radd__')
+        df3 = radd(df1, df2)
+        result = self.executor.execute_dataframe(df3, concat=True)[0]
+        expected = data1 + data2
+        pd.testing.assert_frame_equal(expected, result)
+
+    def testMultiAdd(self):
```

Comment:
Could come up with a better name, I thought this test is about something tests like `df1 + df2 + df3` stuff.